src/index.js

```diff
@@ -29,3 +31,26 @@ httpServer.listen({ port }, () => {
     `Subscriptions ready at ws://localhost:${port}${server.subscriptionsPath}`
   )
 })
+
+//Sequelize usage test
+const dummy_user = {
+  firstName: "Ezequiel",
+  lastName: "Giachero",
+  username: "Eze123",
+  salt: "",
+  password: "pepito"
+}
+
+async function createShowDestroy(user){
```

Comment:
I guess this is a test function, isn't it? If so, make sure to not push it to the repo. If you do need the test function, make sure to add a comment explaining this and why you need it.